.config/s3.config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if (getenv('OBJECTSTORE_S3_HOST')) {
```

Comment:
```suggestion
if (getenv('OBJECTSTORE_S3_BUCKET')) {
```

Reply:
Sure, this makes sense. If `S3_HOST` won't be required, should we have a default value?

Reply:
Unset/empty appears to work based on the current version of https://github.com/nextcloud/server/blob/master/lib/private/Files/ObjectStore/S3ConnectionTrait.php